mimesis/providers/generic.py

```diff
@@ -1,20 +1,10 @@
 import inspect
 
 from mimesis.providers import (Address, BaseProvider, Business, ClothingSizes,
-                               Code, Datetime, Development, File, Food)
-
-# TODO: Resolve importing errors and use importing format as above.
-from .games import Games
-from .hardware import Hardware
-from .internet import Internet
-from .numbers import Numbers
-from .path import Path
-from .personal import Personal
-from .science import Science
-from .text import Text
-from .transport import Transport
-from .units import UnitSystem
-from .cryptographic import Cryptographic
+                               Code, Datetime, Development, File, Food, Games,
```

Comment:
Yes, I could. But current version of generic.py already breaks the rule, doesn't it? Should I refactor all lines with imports? I suppose @lk-geimfari wanted to use more compact view in this case.